deploy/scripts/0006-CarbonVortex-upgrade.ts

```diff
@@ -0,0 +1,20 @@
+import { DeployedContracts, InstanceName, setDeploymentMetadata, upgradeProxy } from '../../utils/Deploy';
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+
+const func: DeployFunction = async ({ getNamedAccounts }: HardhatRuntimeEnvironment) => {
+    const { deployer, bnt, bancorNetworkV3 } = await getNamedAccounts();
+
+    const carbonController = await DeployedContracts.CarbonController.deployed();
+
+    await upgradeProxy({
+        name: InstanceName.CarbonVortex,
```

Comment:
This might not work if the "instance name" was perviously "FeeBurner", no?

Reply:
I've renamed all instances of FeeBurner with CarbonVortex (even in deployments) - it was the easier approach than keeping a legacy FeeBurner interface and contract.

Reply:
Ah, gotcha!